apps/editing-toolkit/editing-toolkit-plugin/block-patterns/src/premium-block-patterns.tsx

```diff
@@ -16,13 +17,17 @@ interface PatternTitleProps {
 	description?: string;
 }
 
+interface ExperimentalEditorSettings extends EditorSettings {
+	__experimentalBlockPatterns: ExperimentalBlockPattern[];
+}
+
 interface ExperimentalBlockPattern {
 	categories: string[];
 	content: string;
 	description: string;
 	isPremium: boolean;
 	name: string;
-	title: string;
+	title: React.ReactNode;
```

Comment:
nit: Use type import `import type { ReactNode } from 'react'` rather than accessing the type off the React object.

Reply:
Done! Does `import React, { ReactNode } from 'react';` also work the same way? (It doesn't show errors for me locally)